packages/cf-component-icon/src/Icon.js

```diff
@@ -1,24 +1,102 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import * as svgs from './svgs.js';
+
+import { createComponent } from 'cf-style-container';
+
+const DEFAULT_HEIGHT = 15.5;
```

Comment:
why 15.5?

Reply:
That's what font-size converts them to. So this is `font-size:1rem`, 1.5rem etc etc.
If you have any better way of converting font-size:some rem to a height. I'm super open to suggestions :)